packages/amplify-graphql-model-transformer/src/resources/rds-model-resource-generator.ts

```diff
@@ -0,0 +1,21 @@
+import * as cdk from '@aws-cdk/core';
+import { TransformerContextProvider } from '@aws-amplify/graphql-transformer-interfaces';
+import { ModelResourceGenerator } from './model-resource-generator';
+import {DatasourceType} from '@aws-amplify/graphql-transformer-core';
+import {DynamoDBModelVTLGenerator, ModelVTLGenerator, RDSModelVTLGenerator} from '../resolvers';
+
+/**
+ * An implementation of ModelResourceGenerator responsible for generated CloudFormation resources
+ * for models backed by an RDS data source
+ */
+export class RdsModelResourceGenerator extends ModelResourceGenerator {
+  protected generatorType = 'DynamoModelResourceGenerator';
+
+  generateResources(ctx: TransformerContextProvider): void {
```

Comment:
Aren't we generating anything here?

Reply:
Yeah but the code wasn't in there to refactor because it got deleted during the merge back, this is just a preliminary PR to get the generators in and I'm still working on the rest